backend-shared/util/operations/types_test.go

```diff
@@ -102,6 +103,167 @@ var _ = Describe("Testing CreateOperation function.", func() {
 			Expect(dbOperationFirst.SeqID).To(Equal(dbOperationSecond.SeqID))
 		})
 	})
+
+	Context("Testing waitForOperationToComplete and IsOperationComplete function.", func() {
+		var ctx context.Context
+		var dbq db.AllDatabaseQueries
+		var k8sClient *sharedutil.ProxyClient
```

Comment:
This can just be `client.Client`, as we're not using the ProxyClient functionality in this test (likewise with removing ProxyClient references below)

Reply:
Done